src/software/ai/motion_constraint/motion_constraint_set_builder.cpp

```diff
@@ -31,45 +31,48 @@ std::set<MotionConstraint> buildMotionConstraintSet(const GameState &game_state,
 }
 
 
-std::set<MotionConstraint> buildMotionConstraintSetFromGameState(
+std::set<TbotsProto::MotionConstraint> buildMotionConstraintSetFromGameState(
     const GameState &game_state)
 {
-    std::set<MotionConstraint> motion_constraints;
-    motion_constraints.insert(MotionConstraint::FRIENDLY_DEFENSE_AREA);
+    std::set<TbotsProto::MotionConstraint> motion_constraints;
+    motion_constraints.insert(TbotsProto::MotionConstraint::FRIENDLY_DEFENSE_AREA);
 
     if (game_state.stayAwayFromBall())
     {
-        motion_constraints.insert(MotionConstraint::HALF_METER_AROUND_BALL);
+        motion_constraints.insert(TbotsProto::MotionConstraint::HALF_METER_AROUND_BALL);
     }
 
     if (game_state.isPenalty())
     {
         if (game_state.isOurPenalty())
         {
-            motion_constraints.insert(MotionConstraint::ENEMY_HALF);
+            motion_constraints.insert(TbotsProto::MotionConstraint::ENEMY_HALF);
         }
         else
         {
             // Is their penalty
-            motion_constraints.insert(MotionConstraint::FRIENDLY_HALF);
+            motion_constraints.insert(TbotsProto::MotionConstraint::FRIENDLY_HALF);
```

Comment:
so according to the robocup rules, our motion constraints are sort of wrong:

5.3.5.3 Throughout the penalty kick procedure, all other robots have to be 1m behind the ball such that they do not interfere the penalty kick procedure.

This should just be tagged as an issue I think

Reply:
interesting, created https://github.com/UBC-Thunderbots/Software/issues/2611